src/SFA.DAS.ApplyService.InternalApi/Controllers/ApplicationController.cs

```diff
@@ -67,6 +67,12 @@ public async Task<ActionResult<List<Domain.Entities.Apply>>> GetOrganisationAppl
             return await _mediator.Send(new GetApplicationsRequest(Guid.Parse(signinId), false));
         }
 
+        [HttpGet("Application/{applicationId}/Contact/{signinId}")]
+        public async Task<ActionResult<Domain.Entities.Apply>> GetOrganisationApplicationForUser(Guid applicationId, string signinId)
```

Comment:
The routing looks a bit off but I can't think of a better one.
Could we change the method name to: GetApplicationByUserId